docs/src/inputs/input/devSandbox/devSandbox.jsx

```diff
@@ -0,0 +1,565 @@
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnBlur').default}
+                >
+                    <ExampleInputOnBlur />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onChange Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onChange</code> event from parent. The <code>value</code> prop is required along with this handler.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnChange').default}
+                >
+                    <ExampleInputOnChange />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onClick Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onClick</code> event from parent.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnClick').default}
+                >
+                    <ExampleInputOnClick />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onFocus Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onFocus</code> event from parent.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnFocus').default}
+                >
+                    <ExampleInputOnFocus />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onKeyDown Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onKeyDown</code> event from parent.
```

Comment:
Maybe reword this slightly?
```
Will call an <code>onKeyDown</code> event handler function provided by the parent.
```

Reply:
Fixed. Please check this [Commit](https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/46c752aa0e781aedba48ebab9e7a168431118aa2)